### DISCUSSION TITLE
Transaction cancelation

### DISCUSSION THREAD

**Discussion by ggepenyan** [blockscout/blockscout#11564] [Q&A]
https://worldchain-mainnet.explorer.alchemy.com/tx/0x035fe59b1720709bc1db4cd7538e58949f48cf6e20c702d54c30c6aaf785a1a2

Is there any way to cancel this transaction? Wherever I wanted to direct the token, it was not received. There was a network issue.

**Comment by ggepenyan** [comment]
Or, where/whom can I ask this question to get help or resolve?

**Comment by sl1depengwyn** [comment]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.
